requirements.txt

```diff
@@ -7,9 +7,7 @@ itsdangerous==0.24
 Jinja2==2.10
 language-selector==0.1
 MarkupSafe==1.0
-pygobject==3.20.0
```

Comment:
Don't know how the changes in `requirements.txt` matters for this PR?

Reply:
@vikasnitd undo this change , it does not relate to PR.